Cargo.toml

```diff
@@ -44,6 +44,7 @@ async-trait = "0.1"
 axum = { version = "0.6", features = ["http2"] }
 bincode = "2.0.1"
 bumpalo = "=3.17.0"
+cairo-annotations = "0.5.2"
```

Comment:
We might consider splitting the cairo-annotations lib with features, so we do not depend on cairo through this.